src/api/app/controllers/webui/cloud/azure/configurations_controller.rb

```diff
@@ -2,8 +2,13 @@ module Webui
   module Cloud
     module Azure
       class ConfigurationsController < WebuiController
-        before_action :require_login
         before_action :set_azure_configuration
+        # TODO: Remove this when we'll refactor kerberos_auth
+        before_action :kerberos_auth
```

Comment:
[As you pointed out](https://github.com/openSUSE/open-build-service/issues/10083#issuecomment-696264073) we need to deal with the `kerberos_mode`. 
I'm not completely sure yet how to deal with `kerberos_mode`, this is just an idea, what's you're take?

**Please note that I didn't test this part locally**

Reply:
looking into the method `kerberos_auth` it should work as "filter". So i guess its ok